girder_jupyter/contents/manager.py

```diff
@@ -410,8 +410,9 @@ def _upload_to_path(self, content, mime_type, format, path):
         parent = self._create_folders('/'.join(folder_path))
 
         if self._is_user(parent):
-            raise web.HTTPError(400, 'Can\'t create file under user.',
-                                reason='Can\'t create file under user.')
+            msg = 'Can\'t create file as a child of a Girder user. Navigate to folder ' \
+                'before creating file.'
```

Comment:
I'd suggest a bit more user-friendly full sentences, something along the lines of:

"The Girder user's home location may only contain folders. Create or navigate to another folder before creating or uploading a file."